scripts/get-resource-index-ref-matrix

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# Checks the Nextstrain production and canary Heroku apps to determine
+# if they are using the same `config.RESOURCE_INDEX` and returns a JSON array
+# of the the commit hashes for the different RESOURCE_INDEX.
+#
+# Intended to be used within the `index-resources.yml` GH Action workflow.
+
+: "${HEROKU_TOKEN:?The HEROKU_TOKEN environment variable is required.}"
+: "${HEROKU_PIPELINE:=38f67fc7-d93c-40c6-a182-501da2f89d9d}"
+
+: "${PROD_APP_NAME:=nextstrain-server}"
+: "${CANARY_APP_NAME:=nextstrain-canary}"
+
```

Comment:
Maybe y'all don't do this in GH Actions for reasons, but IMO for `bash` stuff it's always good to have a `set -euo pipefail` near the top of the script, so errors actually count.

Reply:
Ah, my bad, we do usually do set that at the top! Thanks for catching that!